app/assets/stylesheets/partials/_forms.scss

```diff
@@ -155,3 +155,7 @@ override character count display
         display: block;
     }
 }
+
+.form-group > fieldset > legend > span > h1.heading-large {
+margin-top :0
+}
```

Comment:
```suggestion
}

```

Reply:
Its just styling changes so we do not need any unit tests.